### PR TITLE
Output: Make it shorter

### PR DESCRIPTION
**Pull request by jarifibrahim**
Remove existing menu for choices and add a simpler menu.

Fixes https://github.com/coala-analyzer/coala/issues/1923

### CODE REVIEW

File: coalib/output/ConsoleInteraction.py [hunk 1]
```diff
@@ -490,27 +490,26 @@ def choose_action(console_printer, actions):
...
 
         try:
-            line = format_lines("Please enter the number of the action "
-                                "you want to execute. ")
-            choice = int(input(line))
-            if 0 <= choice <= len(actions):
+            line = format_lines("Please enter the alphabet for the action "
+                                "you want to execute: ")
+            choice = input(line)
+            if choice in {'e', 'o', 'a', 's'}:
```
Comment (sils1297) [inline coalib/output/ConsoleInteraction.py]: hey, you can't just hardcode that, the chars should be generated dynamically from the action texts

**Comment by sils1297** [comment]
https://github.com/coala-analyzer/coala/pull/1926/commits/a19b05a3fc5068bf5b4451600e1353570e73c7b5 needs work

File: coalib/output/ConsoleInteraction.py (hunk 1 above)
Comment (jarifibrahim) [inline coalib/output/ConsoleInteraction.py]: strip `chars` off `actions.desc` ?

Comment (sils1297) [inline coalib/output/ConsoleInteraction.py]: yeah something like that, please discuss the design on the issue, this is not solved with something as trivial as this

**Comment by jarifibrahim** [comment]
@tushar-rishav Yes. It looks better. 
But we first need to figure out a way to map `choices` to `chars`.
I had thought of 
```
action_chars = []
for action in actions
    action_chars.append(action.desc[1]) 
```
but @sils1297 pointed out that this is not a proper approach.

**Comment by tushar-rishav** [comment]
@jarifibrahim Yeah. I noticed that late. I didn't delete my comment because
maybe in future if you come across such constructs, you might wanna
simplify it further. :)
ᐧ

**Comment by jarifibrahim** [comment]
@tushar-rishav Thanks :)

**Comment by abhsag24** [comment]
can't we still use numbers like always ? and just make the text shorter? why does 1 caracter make so much difference?

**Comment by jarifibrahim** [comment]
@abhsag24 I guess we should just stick to the numbers and just add the colors to the output messages (They look super cool [see this](http://imgur.com/6Z2YOYo) )

**Comment by abhsag24** [comment]
could you comment [this](https://github.com/coala-analyzer/coala/pull/1926#issuecomment-199357336) on the issue? you should always document your suggestions and dedcisions, so if say someone wants to know why you made a particular decision they could always see it.

File: coalib/output/ConsoleInteraction.py [hunk 2]
```diff
@@ -490,27 +490,26 @@ def choose_action(console_printer, actions):
...
                 return choice
         except ValueError:
             pass
 
-        console_printer.print(format_lines("Please enter a valid number."))
+        console_printer.print(format_lines("Please enter a valid alphabet."))
```
Comment (Adrianzatreanu) [inline coalib/output/ConsoleInteraction.py]: i think correct is ``letter`` , not ``alphabet``

File: coalib/results/result_actions/ApplyPatchAction.py [hunk 3]
```diff
@@ -30,7 +30,7 @@ def apply(self,
               file_diff_dict,
               no_orig: bool=False):
         """
-        Apply the patch automatically.
+        (a)pply the patch.
```
Comment (Adrianzatreanu) [inline coalib/results/result_actions/ApplyPatchAction.py]: In my opinion, if we were to use this, you should still capitalize the first letter.

File: coalib/results/result_actions/OpenEditorAction.py [hunk 4]
```diff
@@ -24,7 +24,7 @@ def is_applicable(result, original_file_dict, file_diff_dict):
 
     def apply(self, result, original_file_dict, file_diff_dict, editor: str):
         '''
-        Open the affected file(s) in an editor.
+        (o)pen the affected file in an editor.
```
Comment (Adrianzatreanu) [inline coalib/results/result_actions/OpenEditorAction.py]: same here

File: coalib/results/result_actions/ShowPatchAction.py [hunk 5]
```diff
@@ -73,7 +73,7 @@ def apply(self,
               file_diff_dict,
               colored: bool=True):
         '''
-        Print a diff of the patch that would be applied.
+        (s)how patch that would be applied.
```
Comment (Adrianzatreanu) [inline coalib/results/result_actions/ShowPatchAction.py]: and here

File: coalib/results/result_actions/ApplyPatchAction.py (hunk 3 above)
Comment (SanketDG) [inline coalib/results/result_actions/ApplyPatchAction.py]: no, this will appear in a menu for keyboard input. Lowercase look sane here.

**Comment by SanketDG** [comment]
@jarifibrahim Ping. Are you still working on this? A couple of iterations and this should be good to go :)

**Comment by hypothesist** [comment]
@jarifibrahim are you still working on this? Pretty close to wrapping this up.

**Comment by Uran198** [comment]
The corresponding issue was closed, so there is no point in this PR.

**Pull request closed without merging**